docs/documentation/quartz-3.x/how-tos/multiple-triggers.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Multiple Triggers
+---
+
+One nice thing about Quartz is that you can split the job registration from the 
+triggers. One nice use case is being able to split out the JobData for each
+trigger. 
+
+Below, we have two triggers, each with their own set of data, but
+we only had to register the one job.
+
+
+```csharp
+public Task DoSomething(IScheduler schedule, CancellationToken ct)
+{
+    var job = JobBuilder.Create<AnExampleJob>()
+                        .WithIdentity("customer-process", "group")
+                        .Build();
+    
+    var replace = true;
+    var durable = true;
+    await schedule.AddJob(job, replace, durable, ct);
+
+    // Trigger 1
+    var jobData1 = new JobDataMap { { "CustomerId", 1 } };
+    await scheduler.TriggerJob(new JobKey("customer-process", "group"), jobData1, ct);
```

Comment:
no applicable to this example as job types are not part of it, but I also like to have the the key bound to job as `public static readonly JobKey` when applicable, then it's easy to reference `AnExampleJob.Key`

Reply:
Well snap! That's gold.